docs/how-to/internal/x11passthrough.rst

```diff
@@ -0,0 +1,270 @@
+======================
+Docker X11 Passthrough
+======================
+
+The ability to run apps targeting other Linux distributions was added in
+`beeware/briefcase#1603 <https://github.com/beeware/briefcase/pull/1603>`__.
+This how-to explains the details of exposing the host environment to the Docker
+container in which the app runs.
+
+X Window System Background
+--------------------------
+
+Nearly all modern Linux distributions use `Version 11
+<https://www.x.org/releases/X11R7.7/doc/xproto/x11protocol.html>`__ of the X
+Window System Protocol (X11) as the foundation for their graphical displays.
+Notably, though, `Wayland <https://wayland.freedesktop.org/>`__ is gaining
+traction as the successor to X11. Due to reasons of entrenchment, Wayland will
+likely continue providing their X11 compatibility layer to support X
+applications well in to the future.
```

Comment:
```suggestion
Linux distributions use either `X Windows <https://www.x.org/>`__(sometimes called X or
X11) or `Wayland <https://wayland.freedesktop.org/>`__ to manage their graphical
displays. X11 is the older of the two; Wayland maintains compatibility with the `X11
protocol <https://www.x.org/releases/X11R7.7/doc/xproto/x11protocol.html>`__ for
backwards compatibility.
```

Reply:
hmmm...."X Windows" sounds like msft has something to do with this? What about "X Window Manager"?